openmdao/drivers/tests/test_pyoptsparse_driver.py

```diff
@@ -1020,7 +1020,7 @@ def setup(self):
 
                 self.declare_partials('*', '*')
 
-                self.fail_deriv = [2, 4]
+                self.fail_deriv = [3, 5]
```

Comment:
Why did you change which derivs fail on this test?

Reply:
I don't remember why I had to do that, but I just reverted the change and the test passes, so I'll get rid of that change.